lora-phy/src/sx127x/sx1276.rs

```diff
@@ -108,7 +129,48 @@ impl Sx127xVariant for Sx1276 {
         }
         let mut config_3 = radio.read_register(Register::RegModemConfig3).await?;
         config_3 = (config_3 & 0xf3u8) | ldro_agc_auto_flags;
-        radio.write_register(Register::RegModemConfig3, config_3).await
+        radio.write_register(Register::RegModemConfig3, config_3).await?;
+
+        if radio.data.sensitivity_quirk {
+            // apply Errata 2.1: Sensitivity optimization with 500 kHz bandwidth
+
+            if mdltn_params.bandwidth == Bandwidth::_500KHz {
+                let val_1 = 0x02;
+                match mdltn_params.frequency_in_hz {
+                    862..=1020 => {
```

Comment:
Its unclear to me if the ranges in the errata are inclusive. Thoughts?

Reply:
Firstly, please fix the frequency, currently these ranges are presented in kilohertz, but we should be comparing against hertz (just add extra 3 zeroes).

And inclusive is ok, though datasheets don't really say whether it is or isn't...